### PR TITLE
minor SR prep

### PR DESCRIPTION
**Pull request by z64a**

### CODE REVIEW

File: src/77480.c [hunk 1]
```diff
@@ -125,6 +125,7 @@ HitID player_raycast_below(f32 yaw, f32 diameter, f32* outX, f32* outY, f32* out
         ret = hitID;
     }
 
+    //@bug duplicate test -- same as first one. should be +/-?
```
Comment (bates64) [inline src/77480.c:128]:
```suggestion
    // @bug duplicate test -- same as first one. should be +/-?
```

File: src/77480.c [hunk 2]
```diff
@@ -142,6 +143,7 @@ HitID player_raycast_below(f32 yaw, f32 diameter, f32* outX, f32* outY, f32* out
         ret = hitID;
     }
 
+    //@bug duplicate test -- same as second one. should be -/+?
```
Comment (bates64) [inline src/77480.c:146]:
```suggestion
    // @bug duplicate test -- same as second one. should be -/+?
```

File: src/world/area_flo/flo_10/flo_10_4_fountain.c [hunk 3]
```diff
@@ -259,47 +259,53 @@ EvtScript N(EVS_SetupFountain) = {
...
+    Call(SetTexPanner, MODEL_o118, TEX_PANNER_8)
     Thread
         TEX_PAN_PARAMS_ID(TEX_PANNER_8)
         TEX_PAN_PARAMS_STEP(    0,-1500,    0,    0)
         TEX_PAN_PARAMS_FREQ(    0,    1,    0,    0)
         TEX_PAN_PARAMS_INIT(    0,    0,    0,    0)
         Exec(N(EVS_UpdateTexturePan))
     EndThread
+    // surface spread
+    Call(SetTexPanner, MODEL_o119, TEX_PANNER_9)
```
Comment (bates64) [inline src/world/area_flo/flo_10/flo_10_4_fountain.c:308]: would be nice to name the models....

**Review by bates64 — COMMENTED** [review]
great work

File: src/world/area_flo/flo_10/flo_10_4_fountain.c (hunk 3 above)
Comment (z64a) [reply src/world/area_flo/flo_10/flo_10_4_fountain.c:308]: Perhaps, though they are one of the few things we have actual dev names for.

**Review by bates64 — APPROVED** [review]

**Pull request merged**